.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -83,7 +83,13 @@ schedules:
         - /benchmarks/*
       exclude:
         - release/1.x
+    always: true
 
+  - cron: "*/6 1 * * Sat"
+    displayName: Saturday CI, every 6 minutes from 1am
```

Comment:
what about the polar bears ? :)

Reply:
Well, we're running the VMs either way, may as well use them for something 😉